d3ploy/plotter.py

```diff
@@ -204,57 +321,85 @@ def plot_demand_supply_nond3ploy(all_dict, agent_dict, commod, test,
         x, y = get_xy_from_dict(val)
         if top_indx:
-            ax1.bar(x, y, label=key,
-                    edgecolor='none')
+            ax1.bar(x, y, label=key, edgecolor="none")
             prev = y
             top_indx = False
         else:
-            ax1.bar(x, y, label=key,
-                    bottom=prev, edgecolor='none')
+            ax1.bar(x, y, label=key, bottom=prev, edgecolor="none")
             prev = np.add(prev, y)
     ax1.grid()
     ax1.legend()
-    ax1.set_xlabel('Time (month timestep)')
-    ax1.set_ylabel('agents')
+    ax1.set_xlabel("Time (month timestep)")
+    ax1.set_ylabel("Facilities")
 
     if demand_driven:
         if log_scale:
-            ax2.semilogy(*zip(*sorted(dict_demand.items())), '+', color='red',
-                         label='Demand', markersize=size)
+            ax2.semilogy(
+                *zip(*sorted(dict_demand.items())),
+                "+",
+                color="red",
+                label="Demand",
+                markersize=size
+            )
         else:
-            ax2.plot(*zip(*sorted(dict_demand.items())), '+', color='red',
-                     label='Demand', markersize=size)
+            ax2.plot(
+                *zip(*sorted(dict_demand.items())),
+                "+",
+                color="red",
+                label="Demand",
+                markersize=size
+            )
     else:
         if log_scale:
-            ax2.semilogy(*zip(*sorted(dict_demand.items())), '+', color='red',
-                         label='Capacity', markersize=size)
+            ax2.semilogy(
+                *zip(*sorted(dict_demand.items())),
+                "+",
+                color="red",
+                label="Capacity",
+                markersize=size
+            )
         else:
-            ax2.plot(*zip(*sorted(dict_demand.items())), '+', color='red',
-                     label='Capacity', markersize=size)
+            ax2.plot(
+                *zip(*sorted(dict_demand.items())),
+                "+",
+                color="red",
+                label="Capacity",
+                markersize=size
+            )
     if log_scale:
-        ax2.semilogy(*zip(*sorted(dict_supply.items())), 'x', color='c',
-                     label='Supply', markersize=size)
+        ax2.semilogy(
+            *zip(*sorted(dict_supply.items())),
+            "x",
+            color="c",
+            label="Supply",
+            markersize=size
+        )
     else:
-        ax2.plot(*zip(*sorted(dict_supply.items())), 'x', color='c',
-                 label='Supply', markersize=size)
+        ax2.plot(
+            *zip(*sorted(dict_supply.items())),
+            "x",
+            color="c",
+            label="Supply",
+            markersize=size
+        )
     ax2.grid()
-    if commod.lower() == 'power':
-        ax2.set_ylabel('Power (MW)', fontsize=14)
+    if commod.lower() == "power":
+        ax2.set_ylabel("Power (MW)", fontsize=14)
```

Comment:
Nitpicky - but this should probably be MWe, since we aren't concerned with the thermal power.